src/AppWrapper.js

```diff
@@ -0,0 +1,68 @@
+import 'typeface-roboto'
+import 'material-design-icons-iconfont/dist/material-design-icons.css'
+import 'flexboxgrid/css/flexboxgrid.css'
+
+import React from 'react'
+import { Provider } from 'react-redux'
+import { ConnectedRouter } from 'connected-react-router'
+
+import {
+    MuiThemeProvider as Mui3ThemeProvider,
+    createMuiTheme as createMui3Theme,
+} from '@material-ui/core/styles'
+
+import { useConfig } from '@dhis2/app-runtime'
+import { useD2 } from '@dhis2/app-runtime-adapter-d2'
+import { mui3theme } from '@dhis2/d2-ui-core'
+import D2UIApp from '@dhis2/d2-ui-app'
+import { CssReset } from '@dhis2/ui-core'
+
+import { initApi } from './utils/api'
+import history from './utils/history'
+import injectTranslationsToD2 from './utils/injectTranslationsToD2'
+import store from './redux/store'
+import './locales'
+
+import App from './App'
+
+const MUI3Theme = createMui3Theme(mui3theme)
+
+const AppWrapper = () => {
+    const { baseUrl } = useConfig()
+    const { d2 } = useD2({
+        d2Config: {
+            baseUrl: baseUrl + '/api',
```

Comment:
Shouldn't need this, it will pick it up from the `config` automatically.

Reply:
correct, removed